src/Service/ExchangeRatesApi.php

```diff
@@ -90,16 +113,13 @@ protected function getHistoricalExchangeRate(HistoricalExchangeRateQuery $exchan
      */
     public function supportQuery(ExchangeRateQuery $exchangeQuery): bool
     {
-        return true;
+        return $this->options['enterprise'] || 'EUR' === $exchangeQuery->getCurrencyPair()->getBaseCurrency();
```

Comment:
Why EUR? It's USD, isnt it?

exchangeratesapi | USD (free), * (paid)
-- | --

Reply:
My mistake is not changing the readme.

I'll commit today

https://exchangeratesapi.io/documentation/#api_response
```
The delivered exchange rates with the API, are by default connected to the Euro
```